tests/common/reboot.py

```diff
@@ -44,6 +45,13 @@
         "cause": r"'reboot'|Non-Hardware \(reboot",
         "test_reboot_cause_only": False
     },
+    REBOOT_TYPE_SOFT: {
+        "command": "soft-reboot",
+        "timeout": 300,
+        "wait": 120,
```

Comment:
Is this 120s wait by design?

Reply:
soft-reboot doesn't need any tight timing requirement. And it's followed cold reboot wait timing. Any concern?

Reply:
No concerns, I was just curious to know.